lib/membrane_rtsp/server.ex

```diff
@@ -74,6 +74,14 @@ defmodule Membrane.RTSP.Server do
     GenServer.start_link(__MODULE__, config, name: config[:name])
   end
 
+  @doc """
+  Same as `GenServer.stop/2`
+  """
+  @spec stop(pid(), reason :: term(), timeout()) :: :ok
+  def stop(server, reason \\ :normal, timeout \\ :infinity) do
```

Comment:
I think we don't need the `reason` argument. I'd also accept a keyword to make it more future-proof

Reply:
I just copied the signature of `GenServer.stop/2`

Reply:
Yup, that's the point :P GenServer is generic and RTSP server is not. You don't have `RTSPServer.call`. It's not a big deal though, I agree

Reply:
and the keyword would include fields like reason and timeout and also give us ability to add some options later, yes?

Reply:
I don't see a reason to put the reason there :P So it would be just timeout for now

> and also give us ability to add some options later

yeah, without having many default arguments